hpe-kubectl-diagnostic.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
I think we can still call this hpe-log-collector.sh as other one is internal to us.